src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+    private final Color color;
```

Comment:
The 'color' field is already defined in the parent 'Figure' class. Instead of declaring it again, you should use the 'color' from the parent class.